### ISSUE TITLE
Miranda NG appmodule is outdated causing OCR to fail in NVDA

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#14834]
### Steps to reproduce:
1. Open NVDA and Miranda NG
2. Without creating an account, focus the empty contact list
3. Press nvda+r to perform OCR

### Actual behavior:
Following error is written to the log, no OCR performed:
```
IO - speech.speech.speak (22:07:11.242) - MainThread (12908):
Speaking ['Eintrag']
IO - inputCore.InputManager.executeGesture (22:07:13.260) - winInputHook (23052):
Input: kb(laptop):NVDA+r
IO - speech.speech.speak (22:07:13.264) - MainThread (12908):
Speaking ['Erkennung läuft']
ERROR - unhandled exception (22:07:13.273) - Dummy-5 (16776):
Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 232, in 'calling callback function'
  File "contentRecog\uwpOcr.pyc", line 96, in callback
  File "contentRecog\recogUi.pyc", line 155, in _recogOnResult
  File "NVDAObjects\IAccessible\__init__.pyc", line 1456, in setFocus
AttributeError: 'Dynamic_mirandaIMContactListRecogResultNVDAObject' object has no attribute 'IAccessibleObject'
```

### Expected behavior:
OCR is performed without error.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-28089,9fe52071

#### Windows version:
Windows 11 22 H2 update

#### Name and version of other software in use when reproducing the issue:
Miranda NG 0.96.2 stable

#### Other information about your system:
n/a
#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
